third_party/txt/src/txt/paragraph_txt.cc

```diff
@@ -1117,6 +1117,50 @@ void ParagraphTxt::Layout(double width) {
         if (style.has_height_override) {
           // Scale the ascent and descent such that the sum of ascent and
           // descent is `fontsize * style.height * style.font_size`.
+          //
```

Comment:
This is a lot of text inserted into the middle of `Paragraph::Layout`.  Can you break this calculation and the documentation into a separate function?

Reply:
Done!